appserver/web/admin/src/main/java/fish/payara/grizzly/config/admin/cli/SetNetworkListenerConfiguration.java

```diff
@@ -0,0 +1,289 @@
+import javax.inject.Inject;
+import javax.inject.Named;
+import org.glassfish.api.ActionReport;
+import org.glassfish.api.I18n;
+import org.glassfish.api.Param;
+import org.glassfish.api.admin.AdminCommand;
+import org.glassfish.api.admin.AdminCommandContext;
+import org.glassfish.api.admin.CommandLock;
+import org.glassfish.api.admin.ExecuteOn;
+import org.glassfish.api.admin.RestEndpoint;
+import org.glassfish.api.admin.RestEndpoints;
+import org.glassfish.api.admin.RuntimeType;
+import org.glassfish.api.admin.ServerEnvironment;
+import org.glassfish.api.event.EventListener;
+import org.glassfish.api.event.EventTypes;
+import org.glassfish.api.event.Events;
+import org.glassfish.config.support.CommandTarget;
+import org.glassfish.config.support.TargetType;
+import org.glassfish.grizzly.Grizzly;
+import org.glassfish.grizzly.config.dom.NetworkConfig;
+import org.glassfish.grizzly.config.dom.NetworkListener;
+import org.glassfish.grizzly.config.dom.NetworkListeners;
+import org.glassfish.hk2.api.PerLookup;
+import org.glassfish.internal.api.Target;
+import org.glassfish.internal.config.UnprocessedConfigListener;
+import org.glassfish.web.admin.LogFacade;
+import org.jvnet.hk2.annotations.Service;
+import org.jvnet.hk2.config.ConfigBeanProxy;
+import org.jvnet.hk2.config.ConfigCode;
+import org.jvnet.hk2.config.ConfigListener;
+import org.jvnet.hk2.config.ConfigSupport;
+import org.jvnet.hk2.config.SingleConfigCode;
+import org.jvnet.hk2.config.TransactionFailure;
+import org.jvnet.hk2.config.UnprocessedChangeEvent;
+import org.jvnet.hk2.config.UnprocessedChangeEvents;
+
+/**
+ *
+ * @author jonathan coustick
+ * @since 4.1.2.182
+ */
+@ExecuteOn({RuntimeType.DAS, RuntimeType.INSTANCE})
+@TargetType(value = {CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.CONFIG, CommandTarget.DEPLOYMENT_GROUP})
+@Service(name = "set-network-listener-configuration")
+@CommandLock(CommandLock.LockType.NONE)
+@PerLookup
+@I18n("set-network-listener-configuration")
+@RestEndpoints({
+    @RestEndpoint(configBean = NetworkListener.class,
+            opType = RestEndpoint.OpType.POST,
+            description = "Configures a network listener")
+})
+public class SetNetworkListenerConfiguration implements AdminCommand, EventListener {
+
+    private Logger LOGGER = LogFacade.getLogger();
+    private static final ResourceBundle rb = LogFacade.getLogger().getResourceBundle();
+
+    //Parameters
+    @Param(name = "enabled", optional = true)
+    private Boolean enabled;
+
+    @Param(name = "dynamic", optional = true)
+    private Boolean dynamic;
+
+    @Param(name = "address", optional = true)
+    private String address;
+
+    @Param(name = "listenerport", optional = false, alias = "Port")
+    private Integer port;
+
+    @Param(name = "listenerPortRange", alias = "listenerportrange", optional = true)
+    private String portRange;
+    @Param(name = "threadpool", optional = true, alias = "threadPool")
```

Comment:
Can you add an empty line between @Param and the previous instance variable? Just like was done on line 123, 120, etc.